lib/file-detail/file-detail-controller.js

```diff
@@ -27,11 +25,11 @@ module.controller('FileDetailController', function($scope, $state, $stateParams,
   $scope.$parent.selected = {id: $stateParams.fileUid};
   // TODO get profile data directly
   self.profileData = userClient.getProfile();
-  // TODO Return user information directly from cloud
-  $q.when(fileMediatorService.resultMapUser().then(function(resultMap) {
-    $timeout(function(){
-      self.workerMap = resultMap;
-    });
-  }));
+  // FIXME - use modules.user to retrieve workermap
```

Comment:
That's the only part that is not working at the moment. File list is not showing owners. 
This will be resolved once we remove mediator from user module.